cmd/checker/main.go

```diff
@@ -225,6 +226,28 @@ func makeGlobDebugLink(glob string, dir string) string {
 	return fmt.Sprintf("https://www.digitalocean.com/community/tools/glob?comments=true&glob=%s&matches=true%s&tests=", encodedGlob, allTests)
 }
 
+func checkPopularity(ctx context.Context, pckg *packages.Package) {
```

Comment:
do you think we need this function? seems like a lot of redundant checks for npm and git

Reply:
We always want to check the GitHub stars (https://github.com/cdnjs/packages/issues/299). Do you have a smarter way in mind?

Reply:
I think it should better now.

Reply:
aaaah, my bad!! didn't notice that issue - you're right